core/src/test/java/org/elasticsearch/search/SearchModuleTests.java

```diff
@@ -165,12 +165,14 @@ public void testDoubleRegister() {
         };
         expectThrows(IllegalArgumentException.class, registryForPlugin(registersDupePipelineAggregation));
 
-        SearchPlugin registersDupeRescorer = new SearchPlugin() {
-            public List<RescorerSpec<?>> getRescorers() {
-                return singletonList(
-                        new RescorerSpec<>(QueryRescorerBuilder.NAME, QueryRescorerBuilder::new, QueryRescorerBuilder::fromXContent));
-            }
-        };
+        SearchPlugin registersDupeRescorer =
+                () -> {
```

Comment:
How can the compiler know which method this is overriding? It can not, so this can not compile.